sfdc/modules/bulk/client.bal

```diff
@@ -222,6 +227,10 @@ public isolated client class Client {
         } else if (bulkJob.jobDataType == CSV) {
             if (content is io:ReadableByteChannel) {
                 payload = check convertToString(content);
+            } else if (content is stream<string[], error?>) {
+                payload = check convertStringListToString(content);
+            } else if (content is string[][]) {
+                payload = check convertStringListToString(content);
```

Comment:
```suggestion
            } else if (content is stream<string[], error?> || content is string[][]) {
                payload = check convertStringListToString(content);
```

Reply:
Fixed in 2ea4f30e2ed08309e52e4026d558a483d059f7a6 . Thanks